src/style.css

```diff
@@ -0,0 +1,147 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+}
+
+/*#region casd*/
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 33px auto 0;
+}
+
+.card__description {
+  display: flex;
+  flex-direction: column;
+}
+
+.card__title {
+  margin: 40px 16px 4px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  text-align: left;
+  color: #060b35;
+
+}
+
+.card__code {
+  margin: 0 16px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__reviews {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 16px 16px 0;
+  height: 16px;
+}
+
+.number-of-stars {
+  font-size: 10px;
+  display: flex;
+  align-items: flex-start;
+  text-align: right;
+  color: #060b35;
+  font-weight: 400;
+  margin: 0;
+
+}
+
+.card__price-tag {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 24px 16px 0;
+}
+
+.card__price {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  display: flex;
+  align-items: flex-end;
+  color: #616070;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+
+}
+
+.card__sum {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+  color: #060b35;
+}
+
+.card__button {
+  display: block;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-decoration: none;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  background-color: #00acdc;
+  width: 166px;
+  height: 40px;
+  margin: 16px;
+  border-radius: 5px;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
+}
+
+/*#region star*/
```

Comment:
create separate file for each bem block instead of using regions

Reply:
not fixed

Reply:
Of course, that isn't fixed. The first time I made a PR by accident)